config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  get 'pages/home'
+  # get 'pages/home'
   get 'pages/help'
+  get 'pages/about'
+  get '', to: "pages#home"
```

Comment:
`get` with empty string is not ok :). We have `root` path on row 11, please, remove this.